src/Occurrence_indicators_downscaling.Rmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "Occurrence indicators: downscaling"
+author:
+- Damiano Oldoni
+- Add your name
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: true
+    toc_depth: 3
+    toc_float: true
+    number_sections: true
+    theme: yeti
+    df_print: paged
+knit: (function(input_file, encoding) { rmarkdown::render(input_file, encoding = encoding, output_file = paste0("../docs/",sub(".Rmd", ".html", basename(input_file))))})
+---
+
+This document describes how to build indicators of Invasive Alien Species based on occurrence data.
```

Comment:
I would adapt the text here to make clear it is in this one specific on downscaling

Reply:
Right.